driver/src/contracts/stablex_auction_element.rs

```diff
@@ -5,6 +5,8 @@ use ethcontract::{Address, U256};
 use crate::util::CeiledDiv;
 
 pub const AUCTION_ELEMENT_WIDTH: usize = 112;
+// Indexed auction elements have the orderId appened at the end
```

Comment:
nit: I think this should just be a doc comment
```suggestion
/// Indexed auction elements have the orderId appened at the end
```